lib/autoyast.pm

```diff
@@ -783,6 +784,48 @@ sub expand_agama_profile {
     return $profile_url;
 }
 
+=head2 install_jsonnet
+
+ install_jsonnet();
+
+ Install via zypper golang-github-google-jsonnet package in the worker.
+
+=cut
+
+sub install_jsonnet {
+    if (system("sudo", "zypper", "-n", "repos", "systemsmanagement_Agama_Devel") != 0) {
+        system("sudo", "zypper", "-n", "addrepo", "-f", "-G",
+            "https://download.opensuse.org/repositories/systemsmanagement:Agama:Devel/15.6/systemsmanagement:Agama:Devel.repo");
+    }
+
+    if (system("command", "-v", "jsonnet") != 0) {
+        system("sudo", "zypper", "-n", "install", "-f", "golang-github-google-jsonnet");
+    }
+}
+
+=head2 generate_json_profile
+
+ generate_json_profile();
+
+ Return the URL of generated JSON profile
+
+=cut
+
+sub generate_json_profile {
+    my $profile_name = "generated_profile.json";
+
+    install_jsonnet();
+    my $profile_parameters = get_var('AGAMA_PROFILE_PARAMETERS');
```

Comment:
`AGAMA_PROFILE_OPTIONS` so it goes with the same type of naming than we have for Puppeteer.

Another important point here that would make less error-prone these openQA variable, as we can guess with string 'true' of 'false' that they are string or code, we should avoid to pass `--tla-str` and `--tla-code`. Could you please implement a simple parse for that.